umbraco-cms-webapp-redis-cache/azuredeploy.json

```diff
@@ -1,536 +1,549 @@
-               "dependsOn":[
-                  "[concat('Microsoft.Sql/servers/', parameters('serverName'))]"
-               ],
-               "properties":{
-                  "endIpAddress":"0.0.0.0",
-                  "startIpAddress":"0.0.0.0"
-               }
-            }
-         ]
+    },
+    "appName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of azure web app"
+      }
+    },
+    "appServicePlanName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of azure web app service plan"
+      }
+    },
+
+    "appServiceTier": {
+      "type": "string",
+      "allowedValues": [
+        "Standard",
+        "Premium"
+      ],
+      "defaultValue": "Standard",
+      "metadata": {
+        "description": "Choose either Standard or Premium Azure Web Apps pricing tiers. It defaults to Standard"
+      }
+    },
+    "appServiceWorkerSize": {
+      "type": "string",
+      "allowedValues": [
+        "0",
+        "1",
+        "2"
+      ],
+      "defaultValue": "0",
+      "metadata": {
+        "description": "Worker Size( 0=Small, 1=Medium, 2=Large )"
+      }
+    },
+    "appEnableAlwaysOn": {
+      "type": "bool",
+      "metadata": {
+        "description": "Whether Always On should be used for the provisioned Web App."
+      }
+    },
+    "appUse32bitWorkerProcess": {
+      "type": "bool",
+      "defaultValue": true,
+      "metadata": {
+        "description": "Whether 32 bit worker process should be used for the provisioned Web App."
+      }
+    },
+    "dbServerName": {
+      "type": "string",
+      "metadata": {
+        "description": "SQL Azure DB Server name"
+      }
+    },
+    "dbAdministratorLogin": {
+      "type": "string",
+      "metadata": {
+        "description": "SQL Azure DB administrator  user login"
+      }
+    },
+    "dbAdministratorLoginPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Database admin user password"
+      }
+    },
+    "dbName": {
+      "type": "string",
+      "metadata": {
+        "description": "Database Name"
+      }
+    },
+    "dbEdition": {
+      "type": "string",
+      "allowedValues": [
+        "Standard",
+        "Premium"
+      ],
+      "metadata": {
+        "description": "SQL DB appServiceTier : Standard,Premium"
+      }
+    },
+    "dbServiceObjective": {
+      "type": "string",
+      "metadata": {
+        "description": "SQL DB service objective : GUID that can be found in SQL DB Schema on Github. Must be supported by Edition!"
+      }
+    },
+    "dbNonAdminDatabaseUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Non-admin Database User. Must be Unique"
+      }
+    },
+    "dbNonAdminDatabasePassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Non-admin Database User password"
+      }
+    },
+    "storageAccountName": {
```

Comment:
Consider eliminating this and generating this in the variables section. See #944